.github/workflows/conda-build.yaml

```diff
@@ -0,0 +1,54 @@
+name: conda-build
+
+on:
+  push:
+    branches: [ master ]
+    tags: ['*']
+  pull_request:
+    branches: [ master ]
```

Comment:
Are you sure you want this to run on pushes to master? Seems like this should only run on tags?

Reply:
just for now ... just to have the first package available ... as I commented here https://github.com/Quansight/omnisci-datascience-installer/pull/33/files#diff-f611903134d63d57ade017e1b39d13c622ec067b03350a6697cdfc7e69e0931fR42

I will change it in a follow up PR .. also I will change the CI to keep the dependency between both jobs